content/en/serverless/step_functions/installation.md

```diff
@@ -0,0 +1,189 @@
+     5. Enter a filter name. You can choose to name it "empty filter" and leave the *Filter pattern* box blank.
+4. Enable tracing on your Step Functions. Set the environment variable `DD_TRACE_ENABLED` to `true`.
+5. Set up tags. Open your AWS console and go to your Step Functions state machine. Open the *Tags* section and add `env:<ENV_NAME>` and `service:<SERVICE_NAME>` tags. The `env` tag is required to see traces in Datadog. The `service` tag defaults to the state machine's name.
+6. For Node.js and Python runtimes, you can link your Step Function traces to Lambda traces. On the Lambda Task, set the `OutputPath` and `Parameters` keys with the following: 
+
+   ```json
+   "OutputPath": "$.Payload",
+   "Parameters": {
+     "Payload.$": "States.JsonMerge($$, $, false)",
+     "FunctionName": "${lambdaArn}"
+   }
+   ```
+
+   The `JsonMerge` [intrinsic function][6] merges the [Step Functions context object][7] (`$$`) with the original Lambda's input payload (`$`). Fields of the original payload overwrite the Step Functions context object if their keys are the same.
+
+**Example**:
+
+{{< highlight json "hl_lines=4-7" >}}
+"Lambda Read From DynamoDB": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::lambda:invoke",
+      "OutputPath": "$.Payload",
+      "Parameters": {
+        "Payload.$": "States.JsonMerge($$, $, false)",
+        "FunctionName": "${lambdaArn}"
+      },
+      "End": true
+    }
+{{< /highlight >}}
+
+Alternatively, if you have business logic defined in the payload, you could also use the following:
+
+{{< highlight json "hl_lines=8-10" >}}
+"Lambda Read From DynamoDB": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::lambda:invoke",
+      "OutputPath": "$.Payload",
+      "Parameters": {
+        "Payload": {
+          ...
+          "Execution.$": "$$.Execution",
+          "State.$": "$$.State",
+          "StateMachine.$": "$$.StateMachine"
+        },
+        "FunctionName": "${lambdaArn}"
+      },
+      "End": true
+    }
+{{< /highlight >}}
+
+If you have not yet instrumented your Lambda functions to send traces, you can [follow the steps to add the Lambda layer for your preferred runtime][3].
+
+[1]: /logs/guide/forwarder/
+[2]: /logs/guide/forwarder/?tab=cloudformation#upgrade-to-a-new-version
+[3]: /logs/guide/forwarder/?tab=cloudformation#installation
+[4]: /getting_started/integrations/aws/
+[5]: https://app.datadoghq.com/integrations/aws
+[6]: https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-intrinsic-functions.html#asl-intrsc-func-json-manipulate
+[7]: https://docs.aws.amazon.com/step-functions/latest/dg/input-output-contextobject.html
+{{% /tab %}}
+{{< /tabs >}}
+
+
+
+## See your Step Function traces in Datadog
+
+After you have invoked your state machine, go to the [**Trace Explorer**][2] in Datadog. Search for `service:<YOUR_STATE_MACHINE_NAME>` to see all traces associated with that state machine. 
```

Comment:
Search for `service:<YOUR_STATE_MACHINE_NAME>` (if you have not set `service` tag on your state machine) ...